### DISCUSSION TITLE
Does lefthook run in sequence or parallel by default?

### DISCUSSION THREAD

**Discussion by ianzone** [evilmartians/lefthook#626] [Q&A] [answered]
English is not my native language, but I found this description contradictory. 
<img width="575" alt="image" src="https://github.com/evilmartians/lefthook/assets/20043230/277f4e15-9fd5-4286-8d93-80a7f0ccd597">

**Comment by mrexox** [comment, marked as answer]
Lefthook runs sequentially by default. But it will run all commands despite their status. With `piped` option lefthook execution will stop on the first failure. 

I understand why this confuses you. I will update the documentation. Thank you!
